packages/venia-concept/src/components/Input/input.js

```diff
@@ -44,6 +44,11 @@ class Input extends Component {
         helpType: HelpTypes.hint
     };
 
+    componentDidMount() {
+        const { initialValue, onChange } = this.props;
+        initialValue && onChange(initialValue);
```

Comment:
Since we're calling `onChange` directly, let's make it `required` in `propTypes`. Otherwise this could blow up.

Reply:
You are right, thanks